deploy/crds/neutron.openstack.org_v1_ovsnodeosp_cr.yaml

```diff
@@ -7,6 +7,6 @@ spec:
   ovsNodeOspImage: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:7d6ba1bcc4f403733e44cd2a008dafe1887501f7a3c74084566a4f4d1e46250d
   serviceAccount: neutron
   roleName: worker-osp
-  ovnSbRemote: 192.168.25.100
+  ovnSbRemote: tcp:172.30.71.55:6642,tcp:172.30.110.147:6642,tcp:172.30.51.24:6642
```

Comment:
Note that the pattern we've established with these in dev-tools is that operators are deployed by OLM and the CRs are copied into, and customised in, the dev-tools repo. So these changes would also need to end up there when we land this.